charts/rancher-vsphere-csi/101.1.0+up2.6.2-rancher1/Chart.yaml

```diff
@@ -0,0 +1,22 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: vSphere CSI
+  catalog.cattle.io/kube-version: '>= 1.20.0-0 < 1.24.0-0'
```

Comment:
This is not what we expect to see here. The version of chart we're pulling has different constraint (and a lot of other stuff is different):
https://github.com/rancher/vsphere-charts/blob/ba41db5e0814f719722954441edf409c12c45dd4/charts/rancher-vsphere-csi/Chart.yaml#L4

This is probably due to all these `generated-changes/patch` files in `packages` folder that apply updates to charts pulled from `rancher/vsphere-charts` and result in final charts that look incorrect.

I'm not sure how you ended up with `generated-changes/patch` files but the end result is clearly wrong. See another relevant comment here: https://github.com/rancher/charts/pull/2204#discussion_r1019661139.

Reply:
This resulted from adding a patch because I was receiving other errors with `make charts` and thought I needed to patch. A `make patch` patches the chart with updates in the package/generated-changes folder, so I needed to remove that dir and regenerate it _after_ my updates to fix it. I checked the new Chart.yaml and deployment files and they are in line with upstream